src/client/src/orangehrmRecruitmentPlugin/pages/ScheduleInterview.vue

```diff
@@ -0,0 +1,293 @@
+                :key="index"
+                v-model="interviewer.value"
+                :label="$t('recruitment.interviewer')"
+                :show-delete="index > 0"
+                :rules="index === 0 ? rules.interviewer : rules.interviewers"
+                include-employees="onlyCurrent"
+                :required="index === 0"
+                @remove="onRemoveAdmin(index)"
+              />
+              <oxd-button
+                v-if="interviewers.length < 5"
+                icon-name="plus"
+                display-type="text"
+                :label="$t('general.add_another')"
+                @click="onAddAnother"
+              />
+            </oxd-grid-item>
+            <oxd-grid-item>
+              <date-input
+                v-model="schedule.date"
+                :label="$t('general.date')"
+                :placeholder="$t('general.date_format')"
+                :rules="rules.date"
+                required
+              />
+            </oxd-grid-item>
+            <oxd-grid-item>
+              <time-input v-model="schedule.time" :label="$t('general.time')" />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-form-row>
+          <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+            <oxd-grid-item
+              class="orangehrm-save-candidate-page --span-column-2"
+            >
+              <oxd-input-field
+                v-model="schedule.note"
+                :label="$t('general.notes')"
+                type="textarea"
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-divider />
+        <required-text></required-text>
+        <oxd-form-actions>
+          <oxd-button
+            display-type="ghost"
+            :label="$t('general.back')"
+            @click="onBack"
+          />
+          <submit-button :label="$t('general.save')" />
+        </oxd-form-actions>
+      </oxd-form>
+    </div>
+  </div>
+  <interview-attachments
+    :allowed-file-types="allowedFileTypes"
+  ></interview-attachments>
+</template>
+
+<script>
+import {APIService} from '@/core/util/services/api.service';
+import RequiredText from '@/core/components/labels/RequiredText';
+import ProjectAdminAutocomplete from '@/orangehrmTimePlugin/components/ProjectAdminAutocomplete.vue';
+import {navigate} from '@/core/util/helper/navigation';
+import {
+  required,
+  shouldNotExceedCharLength,
+  validDateFormat,
+} from '@ohrm/core/util/validation/rules';
+import InterviewAttachments from '@/orangehrmRecruitmentPlugin/components/InterviewAttachments';
+import TimeInput from '@/core/components/inputs/TimeInput';
+import DateInput from '@/core/components/inputs/DateInput';
+export default {
+  name: 'ScheduleInterview',
+  components: {
+    'date-input': DateInput,
+    'time-input': TimeInput,
+    'interview-attachments': InterviewAttachments,
+    'required-text': RequiredText,
```

Comment:
DateInput, TimeInput, RequiredText are globally available. needless to import and declare in components